src/Xamarin.Android.Build.Tasks/Utilities/PathUtil.cs

```diff
@@ -0,0 +1,289 @@
+using System;
```

Comment:
@grendello so I ported all the RelativePath stuff to Build Tasks.
Had to make some changes as we don't multi target the build tasks so I could not use partial classes for things like DirectorySeperator etc. So had to do a runtime check instead.

Reply:
I guess it's fine until we can start using the relevant BCL bits :)